_posts/2020-10-18-monero-0.17.1.1-released.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: post
+title: "Monero 0.17.1.1 'Oxygen Orion' Point Release"
+summary: Point release containing fix for nodes stuck on block 2210720 
+tags: [releases]
+author: selsta
+image: /blog/assets/2020-09-17-monero-0.17-released/oxygen-orion.png
+---
+
+<div align="center">
+   <img src="{{ page.image }}" width="350px">
+ </div>
+
+# Overview
+
+This is the v0.17.1.1 minor point release of the Monero software.
+
```

Comment:
I would add here something like `**It's highly raccomended to upgrade to this version, since it fixes a bug that caused nodes to be stuck at block 2210720**`